alternator/executor.cc

```diff
@@ -2087,6 +2092,7 @@ class delete_item_operation : public rmw_operation {
         } else {
             _return_attributes = {};
         }
+        _consumed_capacity += 1;
```

Comment:
This line is very confusing, and I'm still not sure what exactly it does. Does it account one **operation**?
Or maybe one **byte**?
If it's one byte, the `+= 1` isn't correct - because two writes of one byte (e.g., in BatchGetItem) need to be counted as two minimal operations - not as one operation of two bytes. I know you didn't implement BatchWriteItem metrics but one day you will.

Reply:
It supposed to be one byte for the case there is no read before write, it's a valid point about batch put item I just as well put there 1KB